extras/codeanalysis/check_copyright_headers.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python3
+#
+# -*- coding: utf-8 -*-
+#
+# check_copyright_headers.py
+#
+# This file is part of NEST.
```

Comment:
```suggestion
# This file is part of NESTML.
```
Or will this change be part of another PR?

Reply:
Yes, we're still waiting for HEP's blessing, perhaps we can discuss it in the NEST meeting today. This PR is only making things consistent for now without introducing that change yet.